src/client/push-client.js

```diff
@@ -65,7 +65,27 @@ let registrationReady = function(registration) {
   });
 };
 
+/**
+ * PushClient is a front end library that simplifies adding push to your
+ * site.
+ */
 export default class PushClient {
+  /**
+   * Constructs a new PushClient.
+   *
+   * If the current browser has a push scription then it will be
+   * obtained in the constructor and sent to the endpointUrl if supplied
+   * and a subscriptionChange event will be dispatched.
+   *
+   * @param {Object} [options] - Options object should be included if you
```

Comment:
I don't think you need any of the square brackets around parameter names (https://esdoc.org/tags.html#object)